mmv1/products/iambeta/WorkloadIdentityPoolProvider.yaml

```
@@ -310,6 +310,7 @@ properties:
           }
           ```
         required: false
+        diff_suppress_func: 'jwksJsonDiffSuppressFunc'
```

```suggestion
        state_func: 'func(v interface{}) string { s, _ := structure.NormalizeJsonString(v); return s }'
```

While this solution works I would recommend we just normalize the value we store in state similar to 
https://github.com/hashicorp/terraform-provider-google-beta/blob/73b34d8a4ecfcdfde6e3a6db5aa2f1a55176326e/google-beta/services/bigquery/resource_bigquery_table.go#L580-L584

---

oh very nice. is that all I need? I'll add your suggestion.